src/index.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
```

Comment:
- [ ] Kindly, you should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
- Typography: font face, size and weight.
- Layout: composition and space between elements. Kindly improve your design to match the original design and don't forget to add the missing `icon`.


||  Original [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))  ||
|-|-----------------------------------|-|
||![go1](https://user-images.githubusercontent.com/57562869/157501449-50bab51c-5494-447a-a950-0c1abbba504d.png)||

||    `Home` page      ||      `Detail` page        ||
|-|----------------------|-| ----------------------|-|
||![bad1](https://user-images.githubusercontent.com/57562869/158005646-5ffb1f13-c095-4852-b3fa-a78183ffe2b7.png) || ![bad2](https://user-images.githubusercontent.com/57562869/158005664-6cc8fa8b-d452-4435-9c8f-8b11d33a4f91.png)||